event-adapter/pkg/cmd/server/start.go

```diff
@@ -51,15 +49,18 @@ func NewCustomMetricsAdapterServerOptions(out, errOut io.Writer) *CustomMetricsA
 	return o
 }
 
-func (o CustomMetricsAdapterServerOptions) Validate(args []string) error {
+// Validate is setting the server
```

Comment:
This comment is really confusing (as well as the comments below). Please make it more informative. I know this is not your code, so you don't have to include much detail. You can just mention that this implementation is a no-op, I think it's there so that EventsAdapterServerOptions implements some interface.